JointAngles[] path_data
float64[] velocity #optional
float64[] acceleration #optional
---
string message
